src/bundle/Resources/translations/menu.en.xliff

```diff
@@ -181,11 +181,6 @@
         <target state="new">Sections</target>
         <note>key: main__admin__sections</note>
       </trans-unit>
-      <trans-unit id="2cc622305036d330c00683520df433ef82f60c9a" resname="main__admin__systeminfo">
-        <source>System Information</source>
-        <target state="new">System Information</target>
-        <note>key: main__admin__systeminfo</note>
```

Comment:
I think Travis is failing because of this removal.
https://res.cloudinary.com/ezplatformtravis/image/upload/v1609231352/screenshots/5feaebf87736f490635340-vendor_ezsystems_ezplatform-admin-ui_features_standard_systeminfo_feature_12_xr95av.png

Reply:
I will check this

Reply:
After merging https://github.com/ezsystems/ez-support-tools/pull/79 it should be ok